src/css/styles.css

```diff
@@ -0,0 +1,147 @@
+/*DEPRECATED
```

Comment:
Can you delete this from the pull request before merging and delete from your master?